.travis.yml

```diff
@@ -46,7 +49,7 @@ jobs:
 
     - stage: "Deploy"
       name: "Build and Push Docker Images"
-      if: branch = master
+      if: branch IN (develop, master) OR (branch =~ /^deploy-.*$/)
```

Comment:
what are examples of `deploy-`? branches?
(I'm mostly familiar with gitflow, which uses `release/version-stuff` -- is `deploy-` similar?

Reply:
I just came up with another branch prefix for my testing (I was using deploy-aws).  I can remove that.

Reply:
it's fine -- just depends on what you want your process to be
in gitflow:
```
 - master                -> production
 - release/*             -> staging 
 - develop               -> ci
 - feature/* or bugfix/* -> review deploy
```